app/src/test/java/mozilla/lockbox/presenter/ItemListPresenterTest.kt

```diff
@@ -102,8 +106,8 @@ open class ItemListPresenterTest {
         override val lockNowClick: Observable<Unit>
             get() = lockNowSelectionStub
 
-        override fun setDisplayName(text: String) {
-            setDisplayNameArgument = text
+        override fun updateAccountProfile(profile: AccountViewModel) {
+            setDisplayNameArgument = profile.displayEmailName
```

Comment:
it might be good to test the `accountName` property in both `ServerProfile.displayName` cases (null vs. non-null).

Reply:
agreed. on my todo list for tomorrow